segmentstore/server/src/main/java/io/pravega/segmentstore/server/SegmentContainer.java

```diff
@@ -27,7 +27,7 @@
 /**
  * Defines a Container for StreamSegments.
  */
-public interface SegmentContainer extends StreamSegmentStore, Container {
+public interface SegmentContainer extends StreamSegmentStore, Container, ContainerEventProcessor {
```

Comment:
Removed this change from `SegmentContainer`. Now the `ContainerEventProcessor` is not exposed through the `SegmentContainer` API, meaning that is basically an internal service of the container itself (it can only be used by other services also instantiated by `StreamSegmentContainer`).